test/fixtures/test-runner/bailout/sequential.test.mjs

```diff
@@ -0,0 +1,12 @@
+
+import { describe, it } from "node:test";
```

Comment:
The fixtures aren't linted, but the style is off in these files (double quotes, inconsistent spacing, etc.).